activist-org/activist
# Issue Resolved : Add missing image alt texts for localization (#405)

### **Description**
Add image alt texts to the en-US.json file to enable localization.

### **Related Issue**
Resolves [#405](https://github.com/activist-org/activist/issues/405)

### **Motivation and Context**
There are alt texts in the codebase that are currently hard-coded and not localized. This pull request aims to fix this issue by adding the missing alt texts to the en-US.json file, allowing for proper localization.

### **Changes Made**

- Added new keys-value pairs to the en-US.json file for each missing alt text.
- Modified the codebase to utilize the new keys using the $t() syntax.

### **How to Test**

- Check that the en-US.json file now contains the added keys and corresponding alt texts.
- Verify that the codebase is updated to use the $t() syntax for alt texts.
- Test the application's functionality and ensure that the images display the correct alt texts based on the selected language.

### **Checklist**

- [ ] Ensured that the code follows the project's coding standards and style guidelines
- [ ] Verified that all tests pass successfully

**_Additional Notes_**

Please review this pull request and provide any feedback or suggestions. I'm happy to address any concerns or make further improvements if needed.
